Client/src/Remote/Endpoints/AuthenticationEndpoint.tsx

```diff
@@ -0,0 +1,20 @@
+import { ISignInUserModel, ISignUpUserModel } from "../../Models/Authentication/ISignUpModel"
+import { IUserAccountModel, IUserAccountRemoteModel, toLocalUserAccountModel } from "../Models/IUserAccountModel"
+
+import { post } from "../RemoteHelper"
+
+const signupRoute = '/signup'
```

Comment:
This is nice. I like the declarations of the routes... These may need to be slightly modified though since I was thinking about renaming the routes to....

api/v1/{routename}

Reply:
thats good, we will taylor the changes as it needs to